tools/cldr-apps/js/src/esm/cldrText.js

```diff
@@ -437,14 +437,14 @@ const strings = {
   special_flagged: "Flagged Items",
   special_forum: "Forum Posts",
   special_forum_participation: "Forum Participation",
-  special_gear: "⚙",
   special_general:
     "Please hover over the sidebar to choose a section to begin entering data. If you have not already done so, please read the <a target='_blank' href='http://www.unicode.org/cldr/survey_tool.html'>Instructions</a>, particularly the Guide and the Walkthrough. You can also use the Dashboard to see all the errors, warnings, and missing items in one place.",
   special_list_emails: "List Email Addresses",
   special_list_users: "List Users",
   special_locales: "Locale List",
   special_lookup: "Look up a code or xpath",
   special_mail: "Notifications (SMOKETEST ONLY)",
+  special_menu: "☰",
```

Comment:
🍔 !
Is a trigram the best way to do this? would it be better as `glyphicons-menu`?

Reply:
maybe, though glyphicons depends on bootstrap which is currently ancient, so reluctant to add more bootstrap dependencies now